src/main/java/com/nvidia/spark/rapids/jni/kudo/KudoHostMergeResult.java

```diff
@@ -61,6 +73,11 @@ public long getDataLength() {
    * @return the cudf table
    */
   public Table toTable() {
+    return toTableStatic(hostBuf, schema, columnInfoList);
+  }
+
+  public static Table toTableStatic(HostMemoryBuffer hostBuf,
```

Comment:
NIT
Not it is an API with 3 separate arguments, so better have some comment for the arguments and what will happen if these arguments mismatch.

Reply:
resolved

Reply:
Good to me